### ISSUE TITLE
Message Action Bar is read out by Windows Narrator *constantly*

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#22581]
### Steps to reproduce

1. Use Windows Narrator
2. Focus on a voice message (for example)
3. Play/seek around in that voice message
4. Note how narrator mentions the download, reply, emoji, etc buttons constantly

#### What did you expect?

For the MAB to be quiet. 

#### What happened instead?

It was noisy. NVDA seems to ignore it, but Narrator works off a premise of "what changed", so this could be an indication that the MAB is over-rendering itself.

### Operating system

Windows 10

### Browser information

Edge

### URL for webapp

local build of develop.element.io

### Will you send logs?

No

**Comment by t3chguy** [comment]
Strange, this should have been avoided by https://github.com/matrix-org/matrix-react-sdk/commit/fd4cdd0dec78d2ea3085ab8b752a394c3f08bea9#diff-8f829517bbb3ff9428ffa49d58ef1eda9c915c81cd3307c6821da69a44ab04dd

**Comment by t3chguy** [comment]
I could not reproduce this on Windows 10 with Edge and Narrator
